pkg/features/kube_features.go

```diff
@@ -125,6 +125,12 @@ const (
 	// Add priority to pods. Priority affects scheduling and preemption of pods.
 	PodPriority utilfeature.Feature = "PodPriority"
 
+	// owner: @resouer @denkensk
+	// alpha: v1.14
+	//
+	// Enable equivalence class for scheduler.
```

Comment:
```suggestion
	// Use concept of equivalence class in scheduler to avoid failure flood when a large group of equivalent pods failed to schedule.
```